apache/docker-entrypoint.sh

```diff
@@ -61,17 +61,22 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		fi
 	fi
 
+	additional_config() {
+	cat <<-'EOPHP'
+	// If we're behind a proxy server and using HTTPS, we need to alert Wordpress of that fact
+	// see also http://codex.wordpress.org/Administration_Over_SSL#Using_a_Reverse_Proxy
+	if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https') {
+		$_SERVER['HTTPS'] = 'on';
+	}
+	
+	EOPHP
+	[ -n "$WP_ALLOW_MULTISITE" ] && echo "define( 'WP_ALLOW_MULTISITE', true );"
```

Comment:
This is useful, but not necessary if `docker-entrypoint.sh` would allow pulling in external snippets (i.e. from /etc/wordpress/*.php).

Why is it not merged yet?